pyproject.toml

```diff
@@ -100,7 +100,7 @@ integration = [
     "scikit-optimize",
     "shap",
     "tensorflow",
-    "torch",
+    "torch<=2.1.2",
```

Comment:
It may be more beneficial to update the CI configuration instead of `pyproject.toml`, aligning with the strategy adopted in  https://github.com/optuna/optuna/pull/4807. By taking this approach, we can proactively avoid unforeseen version conflicts.

Reply:
Thank you for the feedback:)
I will apply your suggestion!